### PR TITLE
MGMTBUGSM-76:Adding clustermetadata to clusterdeployment

### PR DESCRIPTION
**Pull request by tsorya**
Bug 2045067: Adding clustermetadata to clusterdeployment

### CODE REVIEW

File: controllers/agentcluster_controller.go [hunk 1]
```diff
@@ -171,27 +185,28 @@ func (r *AgentClusterReconciler) getControlPlane(ctx context.Context, log logrus
 		return nil, errors.Wrapf(err, "failed to retrieve %s external object %q/%q", obj.GetKind(), key.Namespace, key.Name)
 	}
 
-	var ok bool
-	controlPlane.BaseDomain, ok, err = unstructured.NestedString(obj.UnstructuredContent(), "spec", "dns", "baseDomain")
-	if !ok {
-		log.Infof("Control plane object has no baseDomain, waiting more")
-		return nil, nil
+	fmt.Println("AAAAAAAAAAAA", obj)
```
Comment (avishayt) [inline controllers/agentcluster_controller.go:188]:
```suggestion
	fmt.Println("BBBBBBBBB", obj)
```

Comment (tsorya) [reply controllers/agentcluster_controller.go:188]: as always :(

**Review by avishayt — APPROVED** [review]

**Comment by tsorya** [comment]
/retitle MGMTBUGSM-76:Adding clustermetadata to clusterdeployment

**Pull request merged**